bin/emqx

```diff
@@ -940,6 +940,10 @@ case "${COMMAND}" in
         # set before generate_config
         if [ "${_EMQX_START_DAEMON_MODE:-}" = 1 ]; then
             tr_log_to_env
+            # Ensure console log default is enabled, file log default is disabled.
```

Comment:
isn't it vice versa, console log is disabled by default for daemon and file log is enabled?

Reply:
This only happened in daemon mode.
Don't let console log print to `erlang.log.x` too much, only output them to file(`emqx.log.x`).